lug-helper.sh

```diff
@@ -2313,9 +2312,9 @@ install_game() {
 
         # Run the appropriate installer
         if [ "$install_version" = "native" ]; then
-            lutris && lutris --install "$install_script" &
+            lutris --install "$install_script" &
         elif [ "$install_version" = "flatpak" ]; then
-            flatpak run net.lutris.Lutris && flatpak run --file-forwarding net.lutris.Lutris --install @@ "$install_script" @@ &
+            flatpak run --file-forwarding net.lutris.Lutris --install @@ "$install_script" @@ &
```

Comment:
I noticed you took out the workaround here. Did you decided against the change?

Reply:
yeah seems like it needs some poking upstream and it stuck out to me as being a bit out of scope of the pr so i removed that change in the subsequent commit